com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -509,6 +529,22 @@ private void SendSceneEventData(uint sceneEventId, ulong[] targetClientIds)
         /// <returns>true (Valid) or false (Invalid)</returns>
         internal bool IsSceneNameValid(string sceneName)
         {
+            if (sceneName.Contains("/"))
+            {
+                if (sceneName.Contains(".unity"))
```

Comment:
Again, I don't think it's illegal for a user to have a path that has `.unity` somewhere in it.  Maybe, this could be salvaged by only replacing it if the file ends with `.unity`, not just contains it

Reply:
This is addressed and a test was added to verify the various irregular paths and extension formats will be handled properly.